### ISSUE TITLE
Button smashing still works on Add Level button

### ISSUE THREAD

**Issue by sanjuroj** [mercycorps/TolaActivity#1883]
Jenny's comment on #1331: I'm still having trouble with the "accidentally adding multiple levels" issue. I've done it numerous times just in the first few minutes of playing with it, without trying. (I'm able to intentionally replicate it as well.) If anything, it feels more sensitive than before. It happens when I lightly tap on the trackpad, even when I'm consciously trying to tap only once.

I think you mentioned putting in a fix for that, so wondering if it was merged, or if it's maybe just not doing the trick for some reason.

**Comment by jennyhavah** [comment]
@sanjuroj Can you please describe the fix you already tried, so we don't go down the same path, and/or can build on whatever you did?

**Comment by jennyhavah** [comment]
Also, I am sooooo NOT smashing those buttons! I swear!

**Comment by sanjuroj** [comment]
@jennyhavah I'm referring to anything that results in two mouse clicks in quick succession as a SMASH, not just the frenzied mouse activity of a keyed-up QA tester.  I tried to come up with a pithy acronym for SMASH but don't have the brain power for it at the moment. 

Aaaaannnnnyyyyway, l've removed the old SMASH protection and added a different one.  Because it's hard to tell if you've successfully SMASHed, I've set it so the following appears in the console if the protective measure was triggered:  'Avoided a problem.'.  So if you see that in the console, you **successfully** SMASHed and there shouldn't be an extra blank level field added.  We can remove that logging once QA is done.
